app/helpers/admin/tabbed_nav_helper.rb

```diff
@@ -1,4 +1,77 @@
 module Admin::TabbedNavHelper
+  include Admin::EditionsHelper
+
+  def secondary_navigation_tabs_items(model, current_path)
+    if model.is_a?(Edition)
```

Comment:
This is required so it doesn't call the editions type in the method below 

i.e.  
```
send("#{model.class.model_name.param_key}_nav_items", model, current_path)
 ```

calls 

```
consultation_nav_items(model, current_path)
```
for a consultation.

To avoid this I did think about doing something similar to this https://stackoverflow.com/questions/2393697/look-up-all-descendants-of-a-class-in-ruby

Then adding alias methods for the subclasses, but it seemed a bit overkill tbh.